roles/icpc_host_backup/tasks/main.yml

```diff
@@ -10,7 +10,7 @@
       - perl
       - tcsh
       - fping
-      - ntp
+      # - ntp # this removes systemd-timesyncd, which is not what we want
```

Comment:
hmm, actually do we want to use systemd-timesyncd for the outside connection?    Do we need to configure it?

Reply:
I'm not sure I follow, "outside connection"?

Reply:
These bits,  including talking to the other ntp servers on site, including the clock.
```
pool 0.ubuntu.pool.ntp.org iburst
pool 1.ubuntu.pool.ntp.org iburst
pool 2.ubuntu.pool.ntp.org iburst
pool 3.ubuntu.pool.ntp.org iburst

# Use Ubuntu's ntp server as a fallback.
pool ntp.ubuntu.com

server 10.3.3.208
server 10.3.3.209
server 172.29.1.20
```

Reply:
Hmm, you may be right. Can we merge this as-is and discuss on-site?